app/javascript/schema.json

```diff
@@ -2369,6 +2369,105 @@
               "isDeprecated": false,
               "deprecationReason": null
             },
+            {
+              "name": "interactions",
```

Comment:
The schema change appears to be from the previous commit. I'm ok with a general purpose "adding extended twitter data" commit that covers interaction, profile, media, etc.